### PR TITLE
Added Author Feature and additional description to Contrib section of READ.md

### PR DESCRIPTION
**Pull request by McCune1224**
Features added as mentioned in #6. LMK if you want anything tweaked. Tried to maintain the same syntax used within the project when adding HTML and CSS associated with the author.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -119,6 +119,13 @@ main{
 	z-index: 10;
 }
 
+#deck-author-display {
+    position: absolute;
```
Comment (N-Shar-ma) [inline styles.css:123]: Setting the position again is redundant, so could you please remove this line?

**Review by N-Shar-ma — CHANGES_REQUESTED** [review]
@McCune1224 The code and result look great! Especially appreciate the helpful boilerplate added to the readme :)

It seems you (probably unintentionally) linted all the files due to which there are a lot of white space / tab changes. It's not too big a deal but if you could undo those, it would be nice, so the diff only has the relevant code changes.

One further improvement that just struck me is that it would be nice for the author name to be a link to their GitHub. So instead of a span, could you make it an `a` element with `href` set to the author's GitHub profile? It should look the same as it does now, just an underline on hover to indicate it's a link would be cool.

Sidenote: Adding screenshots showing visual changes is always nice, so please add that when you've made the author's name a link! A screenshot each for non hover and hover state would be nice

**Comment by McCune1224** [comment]
Thanks for the feedback, I'll tweak my editor to do 2 tab instead of 4 tab. I'll also go ahead and change the author name to contain a link to their github.

**Comment by McCune1224** [comment]
Alrighty, made the changes requested:

1.Javascript/HTML:
-  Changed Span to A tag with href to author's github page and updated JS Dom manipulation
-  Fixed 4 tab linting issue, Javascript code should be back to 2 tab now. 
2.CSS: 
-  Added Hover effect for link in css to darken on hover and removed

**Comment by McCune1224** [comment]
# For the CSS on hover the text will be dimmed:

#Non-Hovered
Deck Created non-hover

# Hovered (link displayed in bottom left)
Deck Created hover

**Comment by N-Shar-ma** [comment]
@McCune1224 Looks like you may have forgotten to push? 😅 

Seeing the screenshots, could you make the link have an underline on hover instead of darkening it? Also please make it the same red like the rest of the text and not the default blue.

And in case you haven't already please remove the redundant `position: absolute` from the CSS

**Comment by McCune1224** [comment]
Oops, yeah that would help to push it... Anywho made the additional CSS request to maintain the red and only underline on hover, here's what it looks like now

# non-hover 
Deck Created non-hover

# hover
Deck Created hover

**Comment by N-Shar-ma** [comment]
Merged after a couple minor fixes. Thanks a lot @McCune1224 ! 🚀

**Pull request merged**